### ISSUE TITLE
Specify address book for contacts import

### ISSUE THREAD

**Issue by arjunsarin** [owncloud/contacts#414]
### Steps to reproduce
1. Import contacts from a file (*.ics)
### Expected behaviour

I would like to choose the right address book for the import. Similar to the calendar import.
### Actual behaviour

It is not shown to which address book new contacts are added. Neither can I choose the correct address book.
### Server configuration

**Operating system**: CentOS 6.7 (final)

**Web server:** Apache 2.2

**Database:** MySQL 5.1

**PHP version:** PHP 5.6

**ownCloud version:** ownCloud 9.0.1 (stable)

**Contacts version:** latest master branch (17.05.2016)

**Updated from an older ownCloud or fresh install:** fresh install

**Comment by robin13** [comment]
+1
Also a method to export from one owncloud and import to another owncloud retaining the addressbook assignment.

**Comment by mayerflash** [comment]
This is not really an enhancement, because it was already working under owncloud 8.2 and task 0.7.1
I just installed owncloud 9.1 and I was really disappointed that this function is no longer available.
It's really hard to migrate 500 contacts coming from 3 separate addressbooks...

**Comment by blackpit73** [comment]
yes, we REALLY need this feature. Especially users with large addressbooks cannot move to owncloud like this.

**Comment by christophefromparis** [comment]
I have just updated directly the database to import contacts to the right address book! It works but we have to implement this feature to avoid this manipulation.

**Comment by maky** [comment]
Hi All,
this was working on 8.2 but now in 10 not. We cannot upgrade to version 10 because CONTACTS the best app ever. There is no carddav compatible software on the market which allows to share contacts. I really don't understand how come none of the developers make it available to share contacts for multiple users. This is a winning feature for Owncloud and my business uses it just because of that.

**Comment by mystinen** [comment]
> I have just updated directly the database to import contacts to the right address book! It works but we have to implement this feature to avoid this manipulation.

Could you kindly share this workaround (what to edit in database)? This would be remove a major stumbling block until the functionality is restored! Thanks.
